packages/cli/src/commands/deploy/args.ts

```diff
@@ -21,6 +21,7 @@ export const help = () => `
       link                             Link local directory to a Vercel Project
       login                [email]     Logs into your account or creates a new one
       logout                           Logs out of your account
+      pull                 [filename]  Pull your Project Settings from the cloud
```

Comment:
Deploy uses `[path]`. `link` doesn't specify a sub-argument at all. We should probably at least be consistent here. How about:

```suggestion
      link                 [path]      Link local directory to a Vercel Project
      login                [email]     Logs into your account or creates a new one
      logout                           Logs out of your account
      pull                 [path]      Pull your Project Settings from the cloud
```